### ISSUE TITLE
Invalid credentials for the OSS Index - Credentials work fine on OSSIndex website

### ISSUE THREAD

**Issue by Matheos96** [dependency-check/DependencyCheck#5752]
**Describe the bug**
We are using DependencyCheck in our GitLab CI/CD pipeline and have been for quite some time. Suddenly, out of the blue, today we started getting failures in the job that runs DC. The failure is due to "Invalid credentials for the OSS Index", according to the message. This baffles me a bit though as when I copy the credentials from the CI/CD pipeline variables and paste them into the https://ossindex.sonatype.org/ login page, they work just fine...

**Version of dependency-check used**
CLI 8.2.1

**Log file**
-

**To Reproduce**
Run `dependency-check.sh` with following parameters
```
--project projectName
--scan . 
--format HTML --format JSON 
--prettyPrint 
--cveDownloadWait 0
--ossIndexUsername $OSS_INDEX_USERNAME --ossIndexPassword $OSS_INDEX_PASSWORD
--dotnet $DOTNET_CORE_PATH
```

**Expected behavior**
I don't get an invalid credentials message as the credentials are valid.

**Additional context**
We are using shared runners on gitlab (to which we have no access --> cannot log onto them). I have reran the job 3 times and it has used different runners, same result.

The DC is run inside a Sonar Scanner job (between `sonarscanner begin` and `sonarscanner end`)

**Comment by OrangeDog** [comment]
Same issue with the Maven plugin and an API token.
```xml
<servers>
    <server>
        <id>sonatype-ossindex</id>
        <username>...</username>
        <password>...</password>
    </server>
</servers>
```

However, this account also now fails to log in to their website:

> 401 Error
>
> Authentication failure. It has been logged (ID bf9d8538f71bc1af).

**Comment by Matheos96** [comment]
> However, this account also now fails to log in to their website:
> 
> > 401 Error
> > Authentication failure. It has been logged (ID bf9d8538f71bc1af).

I also got this once. But that was after a period of inactivity on the website. I managed to re-login succesfully.

Furthermore, The pipeline job now went through magically without changes... Not quite sure what is going on

**Comment by OrangeDog** [comment]
See linked issue, but I think they probably fired a bunch of IT staff.

**Comment by Matheos96** [comment]
Yeah I had a look at it earlier. Crazy stuff. Hope they get it sorted. Either way it sounds like this is not a DC issue then rather than something on OSS Index Server side

**Comment by msaubier** [comment]
Would we be missing any critical information in the dependency check report if we disabled the OSS Index analyzer?

**Comment by chadlwilson** [comment]
> Would we be missing any critical information in the dependency check report if we disabled the OSS Index analyzer?

Probably an impossible question to answer objectively as it probably depends on your language environment and your estimation of Sonatype's capabilities as vulnerability researchers and commitment to the OSS community. Anecdotally, in general Java area I find it mostly duplicates the NIST NVD (or at least does not highlight much additional to the NVD), but sometimes they classify the affected product versions differently. It may have a reduced rate of false positives due to the way it classifies libraries compared to DepCheck's use of the NVD data and CPE matching heuristics on library names/identifiers, but that's moot unless you turn off the NVD analyser.

Ive not seen an attempt to do a slightly more objective evaluation.
